.github/workflows/documentation.yaml

```diff
@@ -4,9 +4,6 @@ on:
   push:
     branches:
     - main
-  pull_request:
```

Comment:
yes, perhaps we should remove this from the blueprint as well

Reply:
can be done!